File: ReportsAndTranscripts/GenerateTranscripts/GenerateTranscripts.java
package ReportsAndTranscripts.GenerateTranscripts;

import com.mashape.unirest.http.*;
import com.mashape.unirest.http.exceptions.UnirestException;

public class GenerateTranscripts {
    public static void main(String[] args) throws UnirestException {
        HttpResponse<String> response = Unirest
                .get("https://reports.apexchat.com/Services/ApexChatService.svc/reports/transcripts/detail/3030918")
                .header("apexchat-company", "CompanyName").header("apexchat-username", "Username")
                .header("apexchat-password", "Password")
                .asString();
        System.out.println(response.getBody());
    }
}

File: ReportsAndTranscripts/GenerateTranscripts/GenerateTranscripts.cs
using System;
using RestSharp;

namespace GenerateTranscripts
{
    class GenerateTranscripts
    {
        static void Main(string[] args)
        {
            var client = new RestClient("https://reports.apexchat.com/Services/ApexChatService.svc/reports/transcripts/detail/3030918");
            var request = new RestRequest(Method.GET);
            request.AddHeader("apexchat-company", "CompanyName");
            request.AddHeader("apexchat-username", "Username");
            request.AddHeader("apexchat-password", "Password");
            IRestResponse response = client.Execute(request);
            Console.WriteLine(response.Content);
        }
    }
}
